lvgl/src/lv_core/style.rs

```diff
@@ -64,1144 +64,197 @@ impl From<Opacity> for u8 {
+        const IMG_RECOLOR_OPA = lvgl_sys::lv_style_prop_t_LV_STYLE_IMG_RECOLOR_OPA as u32;
+
+        const LINE_WIDTH = lvgl_sys::lv_style_prop_t_LV_STYLE_LINE_WIDTH as u32;
+        const LINE_DASH_WIDTH = lvgl_sys::lv_style_prop_t_LV_STYLE_LINE_DASH_WIDTH as u32;
+        const LINE_DASH_GAP = lvgl_sys::lv_style_prop_t_LV_STYLE_LINE_DASH_GAP as u32;
+        const LINE_ROUNDED = lvgl_sys::lv_style_prop_t_LV_STYLE_LINE_ROUNDED as u32;
+        const LINE_COLOR = lvgl_sys::lv_style_prop_t_LV_STYLE_LINE_COLOR as u32;
+        //const LINE_COLOR_FILTERED = lvgl_sys::lv_style_prop_t_LV_STYLE_LINE_COLOR_FILTERED as u32;
+        const LINE_OPA = lvgl_sys::lv_style_prop_t_LV_STYLE_LINE_OPA as u32;
+
+        /*Group 5*/
+        const ARC_WIDTH = lvgl_sys::lv_style_prop_t_LV_STYLE_ARC_WIDTH as u32;
+        const ARC_ROUNDED = lvgl_sys::lv_style_prop_t_LV_STYLE_ARC_ROUNDED as u32;
+        const ARC_COLOR = lvgl_sys::lv_style_prop_t_LV_STYLE_ARC_COLOR as u32;
+        //const ARC_COLOR_FILTERED = lvgl_sys::lv_style_prop_t_LV_STYLE_ARC_COLOR_FILTERED as u32;
+        const ARC_OPA = lvgl_sys::lv_style_prop_t_LV_STYLE_ARC_OPA as u32;
+        const ARC_IMG_SRC = lvgl_sys::lv_style_prop_t_LV_STYLE_ARC_IMG_SRC as u32;
+
+        const TEXT_COLOR = lvgl_sys::lv_style_prop_t_LV_STYLE_TEXT_COLOR as u32;
+        //const TEXT_COLOR_FILTERED = lvgl_sys::lv_style_prop_t_LV_STYLE_TEXT_COLOR_FILTERED as u32;
+        const TEXT_OPA = lvgl_sys::lv_style_prop_t_LV_STYLE_TEXT_OPA as u32;
+        const TEXT_FONT = lvgl_sys::lv_style_prop_t_LV_STYLE_TEXT_FONT as u32;
+        const TEXT_LETTER_SPACE = lvgl_sys::lv_style_prop_t_LV_STYLE_TEXT_LETTER_SPACE as u32;
+        const TEXT_LINE_SPACE = lvgl_sys::lv_style_prop_t_LV_STYLE_TEXT_LINE_SPACE as u32;
+        const TEXT_DECOR = lvgl_sys::lv_style_prop_t_LV_STYLE_TEXT_DECOR as u32;
+        const TEXT_ALIGN = lvgl_sys::lv_style_prop_t_LV_STYLE_TEXT_ALIGN as u32;
+
+        /*Group 6*/
+        const RADIUS = lvgl_sys::lv_style_prop_t_LV_STYLE_RADIUS as u32;
+        const CLIP_CORNER = lvgl_sys::lv_style_prop_t_LV_STYLE_CLIP_CORNER as u32;
+        const OPA = lvgl_sys::lv_style_prop_t_LV_STYLE_OPA as u32;
+        const COLOR_FILTER_DSC = lvgl_sys::lv_style_prop_t_LV_STYLE_COLOR_FILTER_DSC as u32;
+        const COLOR_FILTER_OPA = lvgl_sys::lv_style_prop_t_LV_STYLE_COLOR_FILTER_OPA as u32;
+        const ANIM_TIME = lvgl_sys::lv_style_prop_t_LV_STYLE_ANIM_TIME as u32;
+        const ANIM_SPEED = lvgl_sys::lv_style_prop_t_LV_STYLE_ANIM_SPEED as u32;
+        const TRANSITION = lvgl_sys::lv_style_prop_t_LV_STYLE_TRANSITION as u32;
+        const BLEND_MODE = lvgl_sys::lv_style_prop_t_LV_STYLE_BLEND_MODE as u32;
+        const LAYOUT = lvgl_sys::lv_style_prop_t_LV_STYLE_LAYOUT as u32;
+        const BASE_DIR = lvgl_sys::lv_style_prop_t_LV_STYLE_BASE_DIR as u32;
+
+        const PROP_ANY = lvgl_sys::lv_style_prop_t_LV_STYLE_PROP_ANY as u32;
```

Comment:
Need to flesh out which constants were missed in the fork I copied this from and/or which were added/removed post-8.1

Reply:
Doesn't seem like anything relevant is missing, leaving as-is